recipes/saxpy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "saxpy" %}
+{% set version = "1.0.1.dev167" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/saxpy-{{ version }}.tar.gz
+  sha256: 1e223cacd9404f366434be3137224356568d2ef3d4ab88d8412ef7ab91405744
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pbr >=1.9
+    - pip
+    - python >=3.6
+    - setuptools >=17.1
+  run:
+    - numpy
+    - python >=3.6
+
+test:
+  imports:
+    - saxpy
+  # Disabled pip check test because requirements.txt contains a lot of
+  # superfluous packages
```

Comment:
Could not find a tag on GH that has the same code as this PyPI release. The code is quite old but the setup.py there does not have any requirements. Maybe we can ask the author to issue a new stable tag?

Reply:
I'll see what I can do